src/more-infos/more-info-climate.html

```diff
@@ -74,8 +83,10 @@
           <div hidden$='[[computeTargetTempHidden(stateObj)]]'>Target Temperature</div>
           <paper-slider
             id="temp-slider"
+            class$='[[stateObj.attributes.operation_mode]]'
```

Comment:
Please add the class to the parent div. You don't know how web components handle class.